taf/TAF_Runner.py

```diff
@@ -79,7 +79,7 @@ def __init__(self,
             if (state_code.find(',') != -1) and (run_id.find(',') != -1):
```

Comment:
can we just do list(tuple(zip(eval(state_code), eval(run_id)))) without checking if there are comma(s)?

Reply:
This is probably a similar situation. There is much of this code that'd we want to correct and believe me I hope we get the opportunity to. Until now, we're trying to mimic what was in SAS since it's been rather easy to match.